### PR TITLE
feat: Add customProgramDuration field to Programs

### PR DESCRIPTION
**Pull request by gilsantosjulian**
Description
This PR adds a Program Duration Override filed on Programs model, which describes the months in which a Program can takes. This filed override the current value that is has been displayed in the frontend side (Prospectus)

See an example - Right side - 9 months:
[iux-information-systems](https://www.edx.org/micromasters/iux-information-systems?index=product&queryID=079a2394626ce063d46d4e7db87e5d8f&position=1&linked_from=autocomplete&c=autocomplete)

Related Ticket
[WS-3462](https://2u-internal.atlassian.net/browse/WS-3462)

Screenshots

`Programs Panel`

image

`Degrees Panel`

image

`Degree API Response`

image

`Program API Response`

image

### CODE REVIEW

File: course_discovery/apps/api/serializers.py [hunk 1]
```diff
@@ -1807,7 +1807,7 @@ class Meta:
             'micromasters_long_title', 'micromasters_long_description',
             'micromasters_background_image', 'micromasters_org_name_override', 'costs_fine_print',
             'deadlines_fine_print', 'hubspot_lead_capture_form_id', 'additional_metadata',
-            'specializations', 'program_duration_override'
```
Comment (Ali-D-Akbar) [inline course_discovery/apps/api/serializers.py]: Do we not want this data for degrees api?

**Review by Ali-D-Akbar — CHANGES_REQUESTED** [review]
Failing tests. Look into the fact if you need the field in Degree api or not. If yes, be sure to add it in it's serializer.

File: course_discovery/apps/api/serializers.py (hunk 1 above)
Comment (gilsantosjulian) [reply course_discovery/apps/api/serializers.py]: We want it, I updated it to keep it

**Review by Ali-D-Akbar — APPROVED** [review]
The changes indeed look good but please verify your changes on local and make sure this is the intended changes.

**Review by ansabgillani — APPROVED** [review]

**Pull request merged**